rbac/server/api/users.py

```diff
@@ -166,8 +167,17 @@
     # TODO: We have to remove next_id reference entry from task table.
     user_resource = await users_query.delete_user_resource(conn, next_id)
```

Comment:
We don't need to delete_user_resource() function and delete_user_mapping_by_next_id() function anymore because DeleteUser() does that for us.

Reply:
Also need to run linter and formatter.